gum/backend-x86/gumstalker-x86.c

```diff
@@ -2628,6 +2678,17 @@ gum_exec_block_virtualize_branch_insn (GumExecBlock * block,
   }
   else if (op->type == X86_OP_MEM)
   {
+#if GLIB_SIZEOF_VOID_P == 4 && !defined (HAVE_QNX) && defined (HAVE_WINDOWS)
```

Comment:
`!HAVE_QNX` is always true if `HAVE_WINDOWS` is also true:

```suggestion
#if GLIB_SIZEOF_VOID_P == 4 && defined (HAVE_WINDOWS)
```